packages/zowe-explorer/src/extension.ts

```diff
@@ -344,9 +344,65 @@ export async function activate(context: vscode.ExtensionContext): Promise<ZoweEx
     ZoweExplorerExtender.createInstance(datasetProvider, ussFileProvider, jobsProvider);
 
     await SettingsConfig.standardizeSettings();
+    await watchConfigProfile(context);
     return ZoweExplorerApiRegister.getInstance();
 }
 
+async function watchConfigProfile(context: vscode.ExtensionContext) {
+    if (globals.ISTHEIA) {
+        return undefined;
```

Comment:
If ISTHEIA do we have a route to take, I see the pop up message with reload button was removed.  Should we keep that functionality for Theia environment?

Reply:
Yes, it should still be there, good catch.